docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -491,6 +491,95 @@ The following conventions apply to all attributes:
 </TabItem>
 </Tabs>
 
+#### Search decision requirements
+
+- **V1 endpoint**: `POST http://localhost:8080/v1/drd/search`
+- **V2 endpoint**: `POST http://localhost:8080/v2/decision-requirements/search`
+
+<Tabs groupId="search-decision-requirements" defaultValue="input-adjustments" queryString values={
+[
+{label: 'Input adjustments', value: 'input-adjustments'},
+{label: 'Output adjustments', value: 'output-adjustments'},
+]
+}>
+
+<TabItem value='input-adjustments'>
+
+- Request structure changes as outlined in [general changes][].
+  - `searchAfter` has been moved under `page`.
+  - `size` is now the `limit` in the `page` object.
+- Renamed attributes in the `filter` object
+  - `id` - Use `decisionRequirementsId` instead.
+  - `key` of type `int64` - Use `decisionRequirementsKey` of type `string`.
+  - `name` - Use `decisionRequirementsName` instead.
+- Removed attributes in the `filter` object
+  - `resourceName` - Can no longer be used for filtering.
+
+</TabItem>
+
+<TabItem value='output-adjustments'>
+
+- Response structure changes as outlined in [general changes][].
+  - `total` is moved under the `page` object as `totalItems`.
+  - `sortValues` - Use `lastSortValues` in the `page` object instead.
+- Renamed attributes in the objects of the `items` array
+  - `id` - Use `decisionRequirementsId` instead.
```

Comment:
💭 Let's see how this resolves in the filter part, the same will then be true for this field here in the response.